### PR TITLE
Prepare 0.2 release

### PR DESCRIPTION
**Pull request by kinow**
This is a small change with no associated Issue.

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests (why?).
- [x] Appropriate change log entry included.
- [x] No documentation update required.

### CODE REVIEW

**Comment by kinow** [comment]
So far only issue is the missing sourcemaps, which is normal for production to be disabled. Let me just confirm.

image

**Comment by kinow** [comment]
Included a second commit that disables the sourcemaps in production, as per [Vue.js documentation](https://cli.vuejs.org/config/#productionsourcemap).

image

(the other warnings are normal :+1: )

File: vue.config.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ module.exports = {
   indexPath: 'index.html',
   transpileDependencies: ['vuetify'],
   runtimeCompiler: true,
+  productionSourceMap: process.env.NODE_ENV !== 'production',
```
Comment (oliver-sanders) [inline vue.config.js:28]: What does this do?

**Comment by oliver-sanders** [comment]
Tested all good.

**Comment by kinow** [comment]
Two reviews. Would you like to do a final check @hjoliver ? Otherwise I can prepare the release in some minutes.

**Pull request merged**